### PR TITLE
Pyinstaller + Readme Refactor

### PR DESCRIPTION
**Pull request by endurance**
#165 
This branch handles:
- Deletion of old code
- Pyinstaller working (for at least MacOSX)

### CODE REVIEW

File: bin/installer.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (endurance) [inline bin/installer.sh:1]: this file is what we run in order to create our binary :)

You have to run it in different operating system to get the different binaries (run it in windows to get a .exe, run it in linux to get a .bin, etc...)

File: bin/windows/installChocolatey.cmd [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@echo off
```
Comment (endurance) [inline bin/windows/installChocolatey.cmd:1]: this is an easy install command the Chocolatey pacakge manager for Windows users who already have the repo cloned. Read the README for more info.

File: pyproject.toml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 
 [tool.taskipy.tasks]
 test = "python -m unittest src/tests/**/test_*"
+installer = "./bin/installer.sh"
```
Comment (endurance) [inline pyproject.toml:4]: allows us to run `task installer` to run the thing on the right of the equals.

File: requirements.txt [hunk 4]
```diff
@@ -22,8 +22,8 @@ setuptools==58.0.4
...
 rich==12.3.0
 protobuf==3.19.0
-imageio
-imageio-ffmpeg
 pytest==7.1.2
 taskipy==1.10.2
-pytest-cov==3.0.0
\ No newline at end of file
+pytest-cov==3.0.0
+pyinstaller==5.2
```
Comment (endurance) [inline requirements.txt:28]: bring in pyinstaller

File: setup.cfg [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [versioneer]
 VCS = git
 style = pep440-pre
-versionfile_source = freemocap/_version.py
-versionfile_build = freemocap/_version.py
+versionfile_source = src/_version.py
```
Comment (endurance) [inline setup.cfg:4]: freemocap was deleted so moved over to `src`

File: setup.py [hunk 6]
```diff
@@ -26,7 +26,7 @@
 
 here = path.abspath(path.dirname(__file__))
 
-with open(path.join(here, "README.rst"), encoding="utf-8") as readme_file:
```
Comment (endurance) [inline setup.py:29]: README.rst was removed, using README.md now for setup.py

File: src/cameras/save_synchronized_videos.py [hunk 7]
```diff
@@ -5,7 +5,7 @@
 
 from src.cameras.capture.dataclasses.frame_payload import FramePayload
 from src.cameras.persistence.video_writer.video_recorder import VideoRecorder
-from src.freemocap_qt_gui.refactored_gui.state.app_state import APP_STATE
```
Comment (endurance) [inline src/cameras/save_synchronized_videos.py:8]: removed the old gui, so all real code changes are this import change.

**Review by jonmatthis — APPROVED** [review]

**Review by jonmatthis — APPROVED** [review]

**Pull request merged**